### ISSUE TITLE
some problem

### ISSUE THREAD

**Issue by yyfyan** [HRNet/HRNet-Semantic-Segmentation#40]
good job, now I meet some problem when I read the code:
1.why use the: loss = losses.mean
2.the bn_monent=0.01, but the pytorch default use 0.1. 
3.I find the relu(inplace=False), but most network, like resnet, use inplace=True

**Comment by sunke123** [comment]
You can sum the losses of all the pixels.
I think 'mean' is the default approach.
The BN_momentum 0.1 or 0.01 cannot impact the performance.

Using inplace=False is the problem of InplaceABN.
If you using our code of pytorchv1.1, relu(inplace=True) is adopted.

**Comment by yyfyan** [comment]
OK, thanks.

**Comment by yyfyan** [comment]
But the nn.CrossEntropy default is mean. I think the loss.mean is unnecessary. @sunke123
